flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/LazyFromSourcesSchedulingStrategyTest.java

```diff
@@ -377,4 +377,16 @@ private LazyFromSourcesSchedulingStrategy startScheduling(TestingSchedulingTopol
 		schedulingStrategy.startScheduling();
 		return schedulingStrategy;
 	}
+
+	private static List<ExecutionVertexID> mapExecutionVerticesToIDs(
```

Comment:
Maybe `idsFromVertices` so that the tests are shorter.

Reply:
Ok.

Reply:
done.